File: VB/E4485.Module/BusinessObjects/MyWorkflowDefinition.vb
Imports System.Linq
Imports System.Text
Imports DevExpress.ExpressApp.Workflow.Xpo

Namespace E4485.Module.BusinessObjects
    Public Class MyWorkflowDefinition
        Inherits XpoWorkflowDefinition

        Public Sub New(ByVal session As DevExpress.Xpo.Session)
            MyBase.New(session)
        End Sub
        Public Overrides ReadOnly Property CanCompileForDesigner() As Boolean
            Get
                Return False
            End Get
        End Property
    End Class


End Namespace

File: CS/E4485.Module/BusinessObjects/MyWorkflowDefinition.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.ExpressApp.Workflow.Xpo;

namespace E4485.Module.BusinessObjects {
    public class MyWorkflowDefinition : XpoWorkflowDefinition {
        public MyWorkflowDefinition(DevExpress.Xpo.Session session) : base(session) { }
        public override bool CanCompileForDesigner {
            get { return false; }
        }
    }


}
